servers/navigation_server_3d.cpp

```diff
@@ -92,6 +96,10 @@ void NavigationServer3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("process", "delta_time"), &NavigationServer3D::process);
 
 	ADD_SIGNAL(MethodInfo("map_changed", PropertyInfo(Variant::RID, "map")));
+
+#ifdef DEBUG_ENABLED
+	ADD_SIGNAL(MethodInfo("navigation_debug_changed"));
+#endif // DEBUG_ENABLED
```

Comment:
The exposed API shouldn't differ between debug and release builds, so you should keep this always exposed. You can just make it so that it's never emitted in release.